### ISSUE TITLE
VoiceOver use of iOS app fails navigation

### ISSUE THREAD

**Issue by daryanypl** [NYPL-Simplified/Simplified-iOS#808]
A report from a blind user who is using VoiceOver to navigate the SimplyE app:

"
I went and downloaded the app for IOS and had no trouble finding and downloading a book. 

The trouble began when I went to read the book I had gotten. 

I am on a page with the book title, a "cover, image" area, and an area at the bottom of the page that says page one of two.

The book I downloaded is Meditations, by Marcus Aurelius.

There is no back button visible to me with VoiceOver. I had to unload the app from the app switcher and reload to get back to the main page.

Downloading another book and trying to read it produces the same results.

It sounds like the text of the book is in the "cover area" but that is currently not accessible.
"

**Comment by winniequinn** [comment]
This is a known issue and one due to Readium 1.X being fundamentally incompatible with Voiceover. We cannot address this until the renderer is replaced.
